b2sdk/api.py

```diff
@@ -50,7 +55,8 @@ def url_for_api(info, api_name):
 
 
 class Services:
-    """ Gathers objects that provide high level logic over raw api usage. """
+    """Gathers objects that provide high level logic over raw api usage."""
```

Comment:
I know that `"""Text...` is de-facto standard, but it looks less readable than wrapping in spaces